defaults/main.yml

```diff
@@ -4,6 +4,7 @@
 monasca_checks: {}
 monasca_agent_system_only: False
 monasca_agent_git_repo: https://git.openstack.org/openstack/monasca-agent
+monasca_agent_target: monasca-agent.target
```

Comment:
Whats the reason for putting the legacy name in vars and leaving this here?

`s/monasca_agent_target/monasca_agent_systemd_unit_filename`?